src/csharp/Microsoft.Spark/Sql/SparkSession.cs

```diff
@@ -81,6 +81,15 @@ internal SparkSession(JvmObjectReference jvmObject)
             SparkEnvironment.JvmBridge.CallStaticJavaMethod(
                 s_sparkSessionClassName, "clearDefaultSession");
 
+        /// <summary>
+        /// Returns the active SparkSession for the current thread, returned by the builder.
+        /// </summary>
+        /// <returns>SparkSession object</returns>
+        [Since(Versions.V3_0_0)]
+        public static SparkSession GetActiveSession() =>
```

Comment:
Can you remove the comments regarding *ActiveSession

Reply:
I'm sorry, I don't think I understood which comments you want me to remove?

Reply:
Around 20 lines up, around line 64.

Reply:
Done, thanks!

Reply:
Actually if we remove the comments, then we should add the other *ActiveSession APIs.  Can you add them as well?  These APIs will be dependent on #641

Reply:
Got it. Removed.